solidity/test/unit/XERC20.t.sol

```diff
@@ -206,7 +209,7 @@ contract UnitCreateParams is Base {
   }
 
   function testUseLimitsUpdatesLimit(uint256 _limit, address _minter) public {
-    vm.assume(_limit > 1e6);
+    _limit = bound(_limit, 1e6, 1e40);
```

Comment:
`vm.assume` here limits the _`limit` to be > 1e6
and `bound` here will limit it to be >= 1e6 and <1e40. 
Just to point it out the difference